### DISCUSSION TITLE
"Discrete facts" and "inseparable facts"

### DISCUSSION THREAD

**Discussion by wujastyk** [PanditProject/Website#35] [Help]
These sections in PP are odd, as far as a user is concerned.  Could they not be amalgamated into a single "Notes" area?

The reason for putting data into fields is to index, search or format them.  But "facts" don't need to be indexed alphabetically.  And searching across all data fields already works well.  I cannot imagine a scenario where a user would want to do a keyword search limited to "discrete facts."

**Comment by AmirSimantov** [comment]
The "Discrete facts" field supplies a semi-structured way; it is beneficial for presentation and future parsing if we add a new field.

PRESENTATION
it's easier for the eye to scan when the facts are displayed as a list.
For example, in MS Vienna OBVSG Stein & Hügel Collection Indicus 87 ([entity/109271/manuscript](https://www.panditproject.org/entity/109271/manuscript)) the facts are:
- Unclear text
- Crumbled by touch
- The writing inside is well preserved and legible
- Without restoration
- No review was possible
- History of the manuscript: Bühler (?)
- Leather-binding

STRUCTURE
The example above also serves to demonstrate extracting data if we add a field in the future. We can add a "Binding" field; it will be easier to locate and parse values with a given word list (leather-binding, unbound).

Regarding the "Sets of inseparable facts" field, we ask the contributor to use it only if breaking down her text breaks the idea she wants to pass. You can see it as a complete notion or narrative. We are currently using it also when there is a very long name/title (more than 255 characters that the entity title field permits).

You may read more here: [panditproject.org/info/fields-leftover](https://www.panditproject.org/info/fields-leftover).

**Comment by wujastyk** [comment]
Thanks for the clarification.  Personally, I think PP would be clearer without all this detail and choice, and with a simple free-text "note" field.  I don't see any advantage in "discrete facts."  You say it's it's easier for the eye; not to me. I am used to reading paragraphs.  All the serious catalogues I know follow a standard approach to manuscript description in paragraphs, first giving physical description, and following with content description (or vice versa).  For example, see any of the descriptions by the gods of cataloguing, Neil Ker or Andrew Watson, or indeed [this Yale example](https://archive.org/details/catalogueofmedie02bein/page/n435/mode/2up) by Shailor, which exemplifies best practice.

You say you may wish to locate and parse values with a given word list (leather-binding, unbound).  What purpose would this serve?  Especially since a user can search all of PP for the keyword "leather"?  And is it even PP's job to do this?  Surely PP isn't a cataloguing tool, primarily.  Again, I feel that we're adding more and more detailed breakdowns of information into atoms for not much purpose, and perhaps losing sight of the larger goals of revealing relationships between works, owners, authors, families, and teaching lineages.  That's where the real intellectual payoff of PP emerges.  Rather than parsing bindings, let's show our users family trees and maps of manuscript locations, i.e., information that is not available in the catalogues from which PP extracts its information. 

The use of controlled vocabularies is particularly useful in flat-file systems and when applied to critical information like authors, titles, and subjects where uniformity is essential.  This is where controlled cataloguing vocabularies came from, originally.  Now, systems like PP are relational, so the need for controlled vocabs is largely obsolete.
